docs/src/inputs/button/button.jsx

```diff
@@ -1,194 +1,191 @@
-                Outlined
-                    <Header.Subheader>
-                        Buttons can have different border colors.
-                    </Header.Subheader>
-                </Header>
-
-                <ButtonOutlineExample />
-
-                <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                    {require('!!raw-loader!./buttonOutlineExample').default}
-                </Highlighter>
-
-                {/* Relax */}
-                <Header anchor="relax" size="large" style={{ marginTop: '55px' }} sub>
-                    Relax
-                    <Header.Subheader>
-                        A button can have relaxed padding.
-                    </Header.Subheader>
-                </Header>
-
-                <ButtonRelaxExample />
-
-                <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                    {require('!!raw-loader!./buttonRelaxExample').default}
-                </Highlighter>
-
-                {/* Width */}
-                <Header anchor="width" size="large" style={{ marginTop: '55px' }} sub>
-                    Width
-                    <Header.Subheader>
-                        A button can have a fixed width.
-                    </Header.Subheader>
-                </Header>
-
-                <ButtonWidthExample />
-
-                <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                    {require('!!raw-loader!./buttonWidthExample').default}
-                </Highlighter>
+                <MarkdownContainer>
+                    <Typography
+                        className="description"
+                        variant="body1"
+                    >
+                        {description}
+                    </Typography>
+
+                    {/**
+                     * Solid Button
+                     */}
+                    <Heading
+                        anchorLink="solid-button"
+                        variant="h2"
+                    >
+                        Solid Button
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Types
+                    </Typography>
+
+                    <Example
+                        rawCode={require('!!raw-loader!./examples/exampleButtonSolid').default}
+                    >
+                        <ExampleSolid />
+                    </Example>
+
+                    {/**
+                      * Outline Button
+                      */}
+                    <Heading
+                        anchorLink="outline-button"
+                        variant="h2"
+                    >
+                        Outline Button
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Types
+                    </Typography>
```

Comment:
Same here.  I'll refrain from commenting on everyone, but please double-check all of these on this page.  Were you perhaps intending to keep some of the original "Subheader" text, and copied a `<Typography>` element from somewhere but then forgot to update the inner text?

Reply:
I posted a picture above that shows what is below the text "Types."

Reply:
It's following Design's UI docs.

Reply:
Yep, got it.  Resolving.